rasa/core/policies/memoization.py

```diff
@@ -222,12 +239,25 @@ def predict_action_probabilities(
         interpreter: NaturalLanguageInterpreter,
         **kwargs: Any,
     ) -> PolicyPrediction:
+        rule_only_slots = kwargs.get("rule_only_slots")
```

Comment:
let's use proper parameters instead of getting this from kwargs

Reply:
but then I'll change the signature of `predict_action_probabilities`? wouldn't it be breaking change?

Reply:
Nope, as they should already have the `kwargs` param in place which would catch the new param